core/data/src/test/java/uk/gov/gchq/gaffer/data/element/function/ToElementTupleTest.java

```diff
@@ -31,7 +31,7 @@
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertNull;
 
-public class ToElementTupleTest extends FunctionTest {
+public class ToElementTupleTest extends FunctionTest<ToElementTuple> {
```

Comment:
All of the `FunctionTest`s were changed in this PR to be correctly parameterised